backend/routes/usernameRoute.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+
+router.post('/check-username',  async (req, res) => {
+    const {username} = req.body;
+    const findUser = await prisma.user.findUnique({
+        where: {username}
+    })
+
+    const exists = findUser !== null
```

Comment:
for better readability use an if statement here:

if(findUser) {...}

Reply:
const exists = null;
    if (findUser != null) {
        exists == true;
    }
    res.json({exists});
    what do you think about this